infrastructure/lambda-lloyd-george-record-stitch.tf

```diff
@@ -77,6 +77,10 @@ module "lloyd-george-stitch-lambda" {
   resource_id       = module.lloyd-george-stitch-gateway.gateway_resource_id
   http_method       = "GET"
   api_execution_arn = aws_api_gateway_rest_api.ndr_doc_store_api.execution_arn
+
+  memory_size    = 512
+  lambda_timeout = 450
+
```

Comment:
When deployed on ndrd, lambda's time limit extends to 450 seconds. 
However, there seems to be a hard limit for API gateway of 29 secs, so if the lambda runs over 29 sec, frontend will get a 504 Gateway Timeout error.

Therefore, until we find a way to cope with API gateway's timeout limit, increasing the lambda timeout limit here probably wont help.